ci/release/changelogs/next.md

```diff
@@ -4,6 +4,7 @@
 
 - Grid cells can now contain nested edges [#1629](https://github.com/terrastruct/d2/pull/1629)
 - Edges can now go across constant nears, sequence diagrams, and grids including nested ones. [#1631](https://github.com/terrastruct/d2/pull/1631)
+- Avoid unnecessary byte/string conversion [#1672](https://github.com/terrastruct/d2/pull/1672)
```

Comment:
Changelogs should be relevant for users, this one's just a minor internal optimization.

Reply:
Changelog removed